app/Http/Controllers/Table/LocationController.php

```diff
@@ -37,14 +37,24 @@ class LocationController extends TableController
      * @return array
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
+    private $device_types = ['network',
```

Comment:
this should be an enum class.
Also you have inserted it between the function and it's docblock.

Reply:
somewhere in Code must be the list of Device Types defined, but i cannot find them.
I'd like to use this list.
Also i think about a distinct list of Device Types, to reduce the table columns to present device types